Training/Training/Program.cs

```diff
@@ -1,2 +1,90 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine ("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// Training ~ A training program for new joinees at Metamation, Batch- July 2023.
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------
+// Program.cs
+// On a standard chessboard 8 queens have to be placed so that no queen can attack any other.
+// As per the rules of chess, the queen can move horizontally, vertically or in a diagonal.
+// No two queens can be on the same row or column or diagonal.
+// Each solution should display the Queens in 8x8 grid and print all canonical solutions
+// ---------------------------------------------------------------------------------------
+using System.Text;
+
+Console.OutputEncoding = new UnicodeEncoding ();
+int[] sQueenPlace = new int[8];

```

Comment:
Please refer to the style guide doc. Only static members should start with 's'. Make changes wherever applicable.